Crud.Tests/CrudQueryBuilderTest.cs

```diff
@@ -0,0 +1,160 @@
+using System.Collections.Generic;
+using Crud.library.enums;
+using Crud.library.query;
+using Crud.Tests.utils;
+using Newtonsoft.Json;
+using NFluent;
+using NUnit.Framework;
+
+namespace Crud.Tests
+{
+    public class CrudQueryBuilderTest
+    {
+        private CrudQueryBuilder _qb;
+
+        [SetUp]
+        public void Init()
+        {
+            _qb = new CrudQueryBuilder();
+        }
+
+        [Test]
+        public void TestId()
+        {
+            var query = _qb.Id("foo").Build();
+            var result = query["_id"];
+            Check.That(result).IsEqualTo("foo");
+        }
+
+        [Test]
+        public void TestCreatorId()
+        {
+            var query = _qb.CreatorId("foo").Build();
+            var result = query["creatorId"];
+            Check.That(result).IsEqualTo("foo");
+        }
+
+        [Test]
+        public void TestCreatedAt()
+        {
+            var query = _qb.CreatedAt("foo").Build();
+            var result = query["createdAt"];
+            Check.That(result).IsEqualTo("foo");
+        }
+
+        [Test]
+        public void TestUpdaterId()
+        {
+            var query = _qb.UpdaterId("foo").Build();
+            var result = query["updaterId"];
+            Check.That(result).IsEqualTo("foo");
+        }
+
+        [Test]
+        public void TestUpdatedAt()
+        {
+            var query = _qb.UpdatedAt("foo").Build();
+            var result = query["updatedAt"];
+            Check.That(result).IsEqualTo("foo");
+        }
+
+        /*[Test]
```

Comment:
Why is this test commented out? Please restore it since it seems pretty important

Reply:
Fixed